src/Connection.js

```
@@ -0,0 +1,631 @@
+                            propertyProperty() : propertyProperty,
+                    ])
+                    .filter(([_m, _p, type]) => Model.isModel(type))
+                    .map(([containingModel, propertyName, propertyProperty]) => ({
+                        containingModel,
+                        propertyName,
+                        propertyProperty,
+                    })),
+            )
+            .flat()
+            .reduce((accumulator, {containingModel, propertyName, propertyProperty}) => ({
+                ...accumulator,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)